lib/rubocop/config_loader.rb

```diff
@@ -205,8 +205,8 @@ def dirs_to_search(target_dir)
       end
 
       def old_auto_config_file_warning
-        warn Rainbow('Attention: rubocop-todo.yml has been renamed to ' \
-                     "#{AUTO_GENERATED_FILE}").red
+        warn Rainbow('Attention: rubocop-todo.yml has been renamed to ' +
```

Comment:
Probably this is a case we shouldn't change, as consecutive string literals are processed more efficiently by the interpreter. (and this was simply one big string that didn't fit on a single line)